veggieseasons/analysis_options.yaml

```diff
@@ -0,0 +1,9 @@
+include: package:pedantic/analysis_options.yaml
```

Comment:
Please change this to include: package:pedantic/analysis_options.1.7.0.yaml

That means we can adopt newer versions of pedantic's lints on our time schedule, not when new versions of pedantic drop. This will stop false positive travis failures.

Reply:
Er, I've spent too long in dart land. Closing this out